.github/workflows/tools_ci.yml

```diff
@@ -92,3 +92,28 @@ jobs:
       - name: Test (-cstrict)
         if: ${{ matrix.cc == 'msvc' }}
         run: ./v -W -cstrict test-self cmd
+
+  ubuntu-docker-musl:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 121
+    container:
+      image: thevlang/vlang:ubuntu-build
+      env:
+        V_CI_MUSL: 1
+        V_CI_UBUNTU_MUSL: 1
+        VFLAGS: -cc musl-gcc -gc none
+      volumes:
+        - ${{github.workspace}}:/opt/vlang
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Build V
+        run: make -j4 && ./v -cg -o v cmd/v
+      - name: Verify `v test` works
+        run: |
+          ./v cmd/tools/test_if_v_test_system_works.v
+          ./cmd/tools/test_if_v_test_system_works
+      - name: Test
+        run: ./v test-self cmd
+      - name: Test (-cstrict)
+        run: ./v -W -cstrict test-self cmd
```

Comment:
Why add that here 🤔 ?
The tools_ci.yml has jobs that are very fast now.
In contrast, the docker job will run very slowly, and check practically everything.

Reply:
Oh, you do `./v test-self cmd`, not `./v test-self` .

Reply:
![image](https://github.com/vlang/v/assets/26967/4ba9fb1d-c8f1-419f-ac4a-e16dfaacf72d)
That is nice actually: +2min in parallel to all the rest, and a quick fail if there is a problem.